WordPress/src/main/java/org/wordpress/android/ui/stats/NestedScrollViewExt.java

```diff
@@ -0,0 +1,39 @@
+package org.wordpress.android.ui.stats;
+
+import android.content.Context;
+import android.support.v4.widget.NestedScrollView;
+import android.util.AttributeSet;
+import android.widget.ScrollView;
+
+public class NestedScrollViewExt extends NestedScrollView {
```

Comment:
Good call - I followed along the considerations taken for `ScrollViewExt` class. I checked before submitting the PR and it was being used in Stats only, so I guessed it was OK to leave it there too. Also the only reason why this exists seems to be we need to know the user has reached the bottom of the screen to send some analytics tracking, thus overriding `onScrollChanged` was necessary for this implementation. 
We could do something else entirely, like using `scrollView.getViewTreeObserver().addOnScrollChangedListener`  and just get rid of these two classes. But anyway, it works all good like this.